packages/backend/src/webhook/service.ts

```diff
@@ -54,6 +59,40 @@ async function getWebhookEvent(
   return WebhookEvent.query(deps.knex).findById(id)
 }
 
+interface WebhookEventOptions {
+  types?: string[]
+}
+interface DepositOptions extends WebhookEventOptions {
+  depositAccountId: string
+}
+interface WithdrawalOptions extends WebhookEventOptions {
+  withdrawalAccountId: string
+}
+type WebhookByAccountOptions = DepositOptions | WithdrawalOptions
+
+async function getLatestWebhookEventByAccount(
+  deps: ServiceDependencies,
+  options: WebhookByAccountOptions
+): Promise<WebhookEvent | undefined> {
+  const { types } = options
+
+  const query = WebhookEvent.query(deps.knex)
+    .orderBy('createdAt', 'DESC')
+    .first()
```

Comment:
It looks like it doesnt matter (ran it both ways to verify). Although this lead to my realization that `first()` doesnt do `LIMIT 1`, but simply returns an object instead of an array. I thought I could get it to break in cases where it would return multiple but couldnt (I guess it just returns the first object when there are multiple). Still, it seems like changing this to `.limit(1)` (an actual part of the query) here and doing `.first()` at the end (a way to modify what objection returns) would be more correct.